packages/web-frontend/src/components/HierarchyItem.js

```diff
@@ -29,25 +14,10 @@ import SelectedRadioIcon from 'material-ui/svg-icons/toggle/radio-button-checked
 import UnSelectedRadioIcon from 'material-ui/svg-icons/toggle/radio-button-unchecked';
 import SelectedCheckBoxIcon from 'material-ui/svg-icons/toggle/check-box';
 import UnSelectedCheckBoxIcon from 'material-ui/svg-icons/toggle/check-box-outline-blank';
-import CircularProgress from '@material-ui/core/CircularProgress';
 import { ReferenceTooltip } from '@tupaia/ui-components';
 
 export const HierarchyItem = React.memo(
-  ({
-    label,
-    style,
-    nestedMargin,
-    nestedItems,
-    isCheckBox,
-    isSelected,
-    Icon,
-    isLoading,
-    hasNestedItems,
-    info,
-    onClick,
-    dispatch,
-    ...otherProps
-  }) => {
+  ({ label, nestedMargin, nestedItems, isCheckBox, isSelected, info, onClick }) => {
```

Comment:
Because the entity nav hierarchy was diverging from map overlay hierarchy, I split them out rather than trying to maintain both concerns in a single component. It means this component got simpler as it now just deals with the map overlay case